src/main/java/seedu/address/logic/commands/client/AddClientCommand.java

```diff
@@ -27,14 +27,14 @@ public class AddClientCommand extends ClientCommand {
             + ": Adds a client to the address book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
-            + PREFIX_CLIENT_PHONE + "PHONE "
-            + PREFIX_CLIENT_EMAIL + "EMAIL "
+            + PREFIX_MOBILE + "PHONE "
```

Comment:
Should change all the m/PHONE to display m/MOBILE? to keep it consistent so that the user knows why the prefix is m as well?
```suggestion
            + PREFIX_MOBILE + "MOBILE "
```